src/panels/lovelace/editor/hui-element-editor.ts

```diff
@@ -335,6 +335,7 @@ export abstract class HuiElementEditor<T> extends LitElement {
           );
         }
       } else {
+        this._guiSupported = false;
```

Comment:
also a fix for header and footer editors. these were allowing switch back and forth between code editor and nothing when using buttons